lib/index.js

```diff
@@ -104,5 +140,4 @@ export default class Pageres {
 	}
 }
 
-objectAssign(Pageres.prototype, EventEmitter.prototype);
-objectAssign(Pageres.prototype, require('./util'));
+Object.assign(Pageres.prototype, util);
```

Comment:
@thejameskyle I get lots of errors because Flow can't handle mixins this way? Any way I can work around it?